### PR TITLE
replace pkg/errors

### PR DESCRIPTION
**Pull request by zouyee**
Signed-off-by: Zou Nengren <zouyee1989@gmail.com>

### CODE REVIEW

File: btrfs.go [hunk 1]
```diff
@@ -383,29 +383,29 @@ func SubvolDelete(path string) error {
...
+		return fmt.Errorf("not a btrfs filesystem")
 	}
 
 	return nil
 }
 
 func isFileInfoSubvol(fi os.FileInfo) error {
 	if !fi.IsDir() {
-		errors.Errorf("must be a directory")
+		errors.New("must be a directory")
```
Comment (estesp) [inline btrfs.go]: looks like there was a mistake here with no `return`; CI now caught it:
```
Error: ./btrfs.go:402:13: result of errors.New call not used
```

**Review by estesp — CHANGES_REQUESTED** [review]
need to fix the lack of a return

File: btrfs.go (hunk 1 above)
Comment (zouyee) [reply btrfs.go]: Done

**Review by fuweid — APPROVED** [review]
LGTM

**Review by estesp — APPROVED** [review]
LGTM

**Pull request merged**